cfme/configure/access_control.py

```diff
@@ -516,8 +595,18 @@ def create(self, cancel=False):
         """ Create group method
             Args:
                 cancel: True - if you want to cancel group creation,
-                        by defaul group will be created
+                        by default group will be created
+            Throws:
+                RBACOperationBlocked: If operation is blocked due to current user
+                    not having appropriate permissions OR delete is not allowed
+                    for currently selected user
         """
+
+        flash_blocked_msg = VersionPick({
```

Comment:
**OPTIONAL:** It's actually OK here to do a simple ```conditional```. The ```VersionPick``` objects are generally used where we need to defer resolution. Since this is already inside a method that requires instantiation and will receive an appliance prior to this. It's OK to just say if.....else... It will be faster, as there will be no dict created and no pick run.